src/lib.rs

```diff
@@ -1733,15 +1735,15 @@ impl GooseAttack {
                     // Collect all metrics sent by GooseUser threads.
                     self.sync_metrics(&mut goose_attack_run_state, true).await?;
                     // Record last users for users per second graph in HTML report.
-                    //self.graph_data.record_users_per_second(self.metrics.users, Utc::now());
+                    self.graph_data
+                        .record_users_per_second(self.metrics.users, Utc::now());
                     // The load test is fully stopped at this point.
                     self.metrics
                         .history
                         .push(TestPlanHistory::step(TestPlanStepAction::Finished, 0));
                     //self.graph_data.set_stopped(Utc::now());
```

Comment:
`//self.graph_data.set_stopped(Utc::now());`

It seems this isn't used anymore, so can be removed?

Reply:
Done.